atef/bin/scripts.py

```diff
@@ -0,0 +1,70 @@
+"""
+`atef scripts` runs helper scripts.  Scripts may be added over time.
+
+Try:
+
+"""
```

Comment:
try

Reply:
This was to **try** and get the following
```
$ atef scripts

`atef scripts` runs helper scripts.  Scripts may be added over time.

Try:

    $ atef scripts converter_v0 --help
    $ atef scripts pmgr_check --help
```

I got a bit dizzy messing around with all the help text, and didn't realize it doesn't show up for `atef scripts -h`

Reply:
You can accomplish the same without putting the extra word here in the global docstring, right? It could have equivalently been added during `gather_scripts`?

Reply:
Or, if you were trying to mutate the global `__doc__`, you can do that I think but you have to use `global __doc__` instead of `global DESCRIPTION`

Reply:
Either way, I think having the dangling Try literal here decreases readability

Reply:
Oh I see what you mean now.  Yea I can move that without much issue